openedx/core/lib/xblock_builtin/xblock_discussion/utils.py

```diff
@@ -0,0 +1,17 @@
+"""
+Utils for DiscussionXBlock
```

Comment:
@andy-armstrong OK, sounds good. Made the change in https://github.com/edx/edx-platform/pull/13483/commits/ad5f69a9bcd408962eb54f32ab83fdd18394d7d1, and added some tests for the functions in https://github.com/edx/edx-platform/pull/13483/commits/b65a8fe0f89d506e7472ede5ec2eec7e4e297d54.